packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -1059,6 +1060,124 @@ curl -v -X POST \
 }
 ```
 
+#### Add Smart Card Identity Provider
+
+
+Adds a new `Smart Card` type IdP to your organization
+
+##### Request Example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "X509",
+  "status": "ACTIVE",
+  "features": [],
+  "name": "Smart Card IDP Name",
+  "protocol": {
+    "type": "MTLS",
+    "credentials": {
+      "trust": {
+        "revocation": "CRL",
+        "revocationCacheLifetime": 2880,
+        "issuer": "CN=Test Smart Card, OU=Test OU, O=Test O, C=US",
+        "kid": "45dec5ff-8cdc-48c0-85fe-a4869f1753dc"
+      }
+    }
+  },
+  "policy": {
+    "provisioning": {
+      "action":"DISABLED"
+      }
+    },
+    "maxClockSkew":120000,
+    "subject": {
+      "matchType": "EMAIL",
+      "matchAttribute": "",
+      "userNameTemplate": {
+        "template": "idpuser.subjectAltNameEmail"
```

Comment:
We should document the possible values that are supported. The UI also allows for custom expressions to extract the certificate attribute to be matched.